README.md

```diff
@@ -195,59 +157,142 @@ Options for `upstream.yaml`
 ChartMetadata:
   kubeVersion: '>=1.21-0'
-  icon: https://www.kubewarden.io/images/icon-kubewarden.svg
 ```
 
-### Git Repo
+#### Example: Git Repo
+
 ```yaml
----
 GitRepo: https://github.com/kubewarden/helm-charts.git
 GitBranch: main
 GitSubdirectory: charts/kubewarden-controller
 Vendor: SUSE
 DisplayName: Kubewarden Controller
 ChartMetadata:
   kubeVersion: '>=1.21-0'
-  icon: https://www.kubewarden.io/images/icon-kubewarden.svg
 ```
 
-### GitHub Release
+#### Example: Github Release
+
 ```yaml
----
 GitRepo: https://github.com/kubewarden/helm-charts.git
 GitHubRelease: true
 GitSubdirectory: charts/kubewarden-controller
 Vendor: SUSE
 DisplayName: Kubewarden Controller
 ChartMetadata:
   kubeVersion: '>=1.21-0'
-  icon: https://www.kubewarden.io/images/icon-kubewarden.svg
 ```
+
+
+### `app-readme.md`
+
+`app-readme.md` is a brief description of the app and how to use it. It
+is best to keep it short since the longer `README.md` in your chart will be
+displayed in the UI as detailed description.
+
+
+### `questions.yaml`
+
+`questions.yml` defines a set of questions to display on the chart's
+installation page. It allows users to answer them and configure the chart
+using the UI instead of modifying the chart's values file directly. You
+can find a reference for `questions.yaml`
+[here](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/helm-charts-in-rancher/create-apps#question-variable-reference).
+
+#### Example
+
+```yaml
+questions:
+- variable: password
+  default: ""
+  required: true
+  type: password
+  label: Admin Password
+  group: "Global Settings"
+- variable: service.type
+  default: "ClusterIP"
+  type: enum
+  group: "Service Settings"
+  options:
+    - "ClusterIP"
+    - "NodePort"
+    - "LoadBalancer"
+  required: true
+  label: Service Type
+  show_subquestion_if: "NodePort"
+  subquestions:
+  - variable: service.nodePort
+    default: ""
+    description: "NodePort port number (to set explicitly, choose port between 30000-32767)"
+    type: int
+    min: 30000
+    max: 32767
+    label: Service NodePort
+```
+
+
+## Other Information
+
+### Hidden Charts
+
+Hidden charts have packages with `Hidden: true` in their `upstream.yaml`. When
+`Hidden: true` is present in a package's `upstream.yaml`, `partner-charts-ci`
+ensures that all of its chart versions have the `catalog.cattle.io/hidden`
+annotation set. This annotation causes the Rancher UI to not show the chart.
+If you want to hide a chart, use the `partner-charts-ci hide` subcommand.
+Simply setting `Hidden: true` in `upstream.yaml` will not hide existing chart
+versions.
+
+
+### Featured Charts
+
+In the `Apps > Charts` view, the Rancher UI may have a set of tiles at the top
+featuring several charts. These are the "featured" charts. Charts that are
+featured have the `catalog.cattle.io/featured` annotation set on their latest
+chart version. You can see and control featured charts using the
+`partner-charts-ci feature` subcommands.
```

Comment:
Suggest adding: This annotation is reserved for preferred partners that SUSE has agreed to highlight in the Apps > Charts marquee tiles.